app/__init__.py

```diff
@@ -1,48 +1,31 @@
 from flask import request
 from flask_api import FlaskAPI
-from app.request_lookup import get_results
+from app.request_lookup import get_results, get_indexes
 
 app = FlaskAPI(__name__)
 
 
 @app.route('/', methods=['GET'])
 def root():
-    return "</br>/country/ or </br>/industry/ or </br>/occupation/ or </br>"
+    indexes = get_indexes()
 
-
-@app.route('/status', methods=['GET'])
-def status():
-    return "OK"
+    return dict(indexes=indexes)
 
 
-@app.route('/country/', methods=['GET'])
-def country_search():
+@app.route('/<index>/', methods=['GET'])
+def search(index):
     query = request.args.get('q')
+    language = request.args.get('language', 'en')
     if query:
-        countries = get_results('countries', 'country', query)
-        return dict(countries=countries, count=len(countries))
+        results = get_results(index, language, query)
```

Comment:
Me being picky but `language` can be defined after the `if` as it is not needed elsewhere.